src/core/server/server.ts

```diff
@@ -291,7 +294,7 @@ export class Server {
 
   public async start() {
     this.log.debug('starting server');
-    const startTransaction = apm.startTransaction('server_start', 'kibana_platform');
+    const startTransaction = apm.startTransaction('server-start', kibana-platform);
```

Comment:
```suggestion
    const startTransaction = apm.startTransaction('server-start', 'kibana-platform');
```

Reply:
Don't you just love a good "replace all" 🤦🏻